armonik/variables.tf

```
@@ -151,6 +151,21 @@ variable "control_plane" {
     hpa                  = any
     default_partition    = string
     service_account_name = string
+    #conf
+    conf = optional(list(object({
+      env        = optional(map(string), {})
+      env_secret = optional(set(string), [])
+      mount_secret = optional(map(object({
+        secret = string
+        path   = string
+        mode   = string
+      })), {})
+      env_configmap = optional(set(string), [])
+      env_from_configmap = optional(map(object({
+        configmap = string
+        field     = string
+      })), {})
+    })), [])
```

Even though you don't need it yet, I would recommend to also add the other configurations would talked about in the AEP:

```suggestion
    conf = optional(list(object({
      env        = optional(map(string), {})
      env_configmap = optional(set(string), [])
      env_from_configmap = optional(map(object({
        configmap = string
        field     = string
      })), {})
      env_secret = optional(set(string), [])
      env_from_secret = optional(map(object({
        secret = string
        field  = string
      })), {})
      mount_configmap = optional(map(object({
        configmap = string
        path      = string
        subpath   = optional(string)
        mode      = optional(string, "644")
        items = optional(map(object({
          field = string
          mode  = optional(string)
        })))
      })), {})
      mount_secret = optional(map(object({
        secret  = string
        path    = string
        subpath = optional(string)
        mode    = optional(string, "644")
        items = optional(map(object({
          field = string
          mode  = optional(string)
        })))
      })), {})
    })), [])
```